sdk/storage/azure-storage-files-datalake/CHANGELOG.md

```diff
@@ -37,6 +37,7 @@
 - Changed `DataLakeFileClient::Flush`'s `endingOffset` parameter's name to `position`.
 - Renamed `DataLakePathClient::GetAccessControls` to `DataLakePathClient::GetAccessControlList`.
 - Removed unused parameters, options, results and functions in protocol layer.
+- Removed `c_` for constants and renamed to pascal format.
```

Comment:
I dont think you need this at all. You just changed variables in `Details` namespace. We don't need to update changelog if we only change Details stuff, right?

Reply:
I'm OK with either way.